web/src/js/components/feedback/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+export default class Feedback extends React.Component {
+    constructor() {
+        super()
+        this.reactivate = this.reactivate.bind(this);
+    }
+
+    reactivate(e) {
+        e.preventDefault();
+
+        var data = {
+            id: this.props.params['id'],
+            code: this.props.params['code'],
+            status: this.props.params['completed'] == 'yes' ? 2 : -1
+        };
+
+        // @TODO this should be done on page view, but only once!
+        fetch('/api/feedback', {
```

Comment:
I'm a bit concerned that a public endpoint can cause an email to be sent. Can we solve this in a better manner to avoid spam?